Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -1,20 +1,40 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultEditRequest {
+public class BTPayPalVaultEditRequest {
 
+    var riskCorrelationID: String?
+    let hermesPath: String = "v1/paypal_hermes/generate_edit_fi_url"
     private let editPayPalVaultID: String
-    private let merchantAccountID: String?
+
+    // MARK: - Static properties
+
+    static let callbackURLHostAndPath: String = "onetouch/v1"
 
     /// Initializes a PayPal Edit Request for the edit funding instrument flow
     /// - Parameters:
     ///   - editPayPalVaultID: Required: The `edit_paypal_vault_id` returned from the server side requests,
     ///   `gateway.payment_method.find("payment_method_token")` or `gateway.customer.find("customer_id")`
     ///   - merchantAccountID: Optional: ID of the merchant account; if one is not provided the default will be used
     /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-    public init(editPayPalVaultID: String, merchantAccountID: String? = nil) {
+    public init(editPayPalVaultID: String) {
         self.editPayPalVaultID = editPayPalVaultID
-        self.merchantAccountID = merchantAccountID
+    }
+
+    public func parameters() -> [String: Any] {
```

Comment:
Instead of having `riskCorrelationID` be an object we set, we can instead pass it into this method:
```suggestion
    public func parameters(riskCorrelationID: String) -> [String: Any] {
```